zmon_aws_agent/main.py

```diff
@@ -193,11 +193,6 @@ def main():
         application_entities = aws.get_apps_from_entities(apps, infrastructure_account, region)
 
         if args.postgresql_user and args.postgresql_pass:
-            postgresql_clusters = zmon_client.get_entities({
```

Comment:
@mohabusama please double check, but I don't think it makes sense to fetch them here... we discovered them right before...